# This expresses the speeds of the kart in miles/hour
float32 actual_speed
float32 target_speed